R/histogramForRaster.R

```diff
@@ -0,0 +1,35 @@
+#' Histogram for Raster Shiny Module
+#'
+#' @description This creates UI for shiny module creating histogram of data contained in raste
+#'
+#' @param id An ID string that corresponds with the ID used to call the module server function
+#'
+#' @export
```

Comment:
please be sure to explicitly state any imports from other packages, and ensure all imported packages are listed in `DESCRIPTION`:

```r
#' @importFrom shinycssloaders withSpinner
#' @importFrom shiny h4 NS
#' @importFrom shinydashboard box
```

Reply:
need to add `shinycssloaders` to Imports in DESCRIPTION

Reply:
also, please add an explict `@rdname` tag to the documentation. We've been putting both ui+server portions of each shiny module in the same help file. *I.e.*:

```r
#' @rdname histrogramForRaster
#' @rdname histrogramForRasterUI <- function(id) {...}
```

and

```r
#' @rdname histrogramForRaster
#' @rdname histrogramForRaster <- function(input, output, session, raster) {...}
```

Reply:
Added separate task to set rdnames for other modules as well. https://github.com/PredictiveEcology/SpaDES.shiny/issues/46